VDA-XP-Reward/README.md

```diff
@@ -0,0 +1,67 @@
+# Contract Explain
+## Description
+This contract receive signed off chain data (including Verida XP points) and use that to provide a reward to the address in the input parameters. The contract owner set and update XP to token conversion rate.
+
+The key objectives are:
+
+1. Every month, VDA is deposited into the smart contract as reward tokens
```

Comment:
Update this to match the updated issue (we changed the spec slightly)

Reply:
I just copied this description from the github:
https://github.com/verida/blockchain-contracts/issues/152

Let me know how to update this.

Reply:
That issue has been updated since you copied, so just re copy and paste

Reply:
Updated